features/step_definitions/restrict-pr.js

```diff
@@ -0,0 +1,157 @@
+            rootDir,
+            table,
+            shouldNotDeletePipeline: false
+        });
+
+        this.pipelines[rootDir] = {
+            pipelineId: this.pipelineId
+        };
+    }
+);
+
+When(
+    /^a branch is created for test_branch on "([^"]*)" organization$/,
+    {
+        timeout: TIMEOUT
+    },
+    async function step(sourceOrg) {
+        const postfix = new Date().getTime().toString();
+        const sourceBranch = `test-branch-${postfix}`;
+
+        this.sourceOrg = sourceOrg;
+        this.sourceBranch = sourceBranch;
+
+        await github
+            .createBranch(this.sourceBranch, this.sourceOrg, this.repoName)
+            .catch(() => Assert.fail('Failed to create branch.'));
+    }
+);
+
+When(
+    /^a new file is added to the "([^"]*)" directory$/,
+    {
+        timeout: TIMEOUT
+    },
+    async function step(rootDir) {
+        await github
+            .createFile(this.sourceBranch, this.sourceOrg, this.repoName, rootDir)
+            .catch(() => Assert.fail('Failed to create file.'));
+    }
+);
+
+When(
+    /^a pull request is opened from the "([^"]*)" organization$/,
+    {
+        timeout: TIMEOUT
+    },
+    async function step(sourceOrg) {
+        await github
+            .createPullRequest(`${sourceOrg}:${this.sourceBranch}`, this.targetBranch, this.repoOrg, this.repoName)
+            .then(({ data }) => {
+                this.pullRequestNumber = data.number;
+            })
+            .catch(() => Assert.fail('Failed to create the Pull Request.'));
+    }
+);
+
+Then(
+    /^the PR job of "([^"]*)" is triggered because it is not restricted$/,
+    {
+        timeout: TIMEOUT
+    },
+    async function step(rootDir) {
+        const build = await sdapi.searchForBuild({
+            instance: this.instance,
+            pipelineId: this.pipelines[rootDir].pipelineId,
+            jobName: this.jobName,
+            pullRequestNumber: this.pullRequestNumber,
+            jwt: this.jwt
+        });
+
+        const jobs = await sdapi.findJobs({
+            instance: this.instance,
+            pipelineId: this.pipelines[rootDir].pipelineId,
+            jwt: this.jwt
+        });
+
+        const jobData = jobs.body;
+        const prJobName = `PR-${this.pullRequestNumber}:${this.jobName}`;
+        const job = jobData.find(j => j.name === prJobName);
+
+        Assert.equal(build.jobId, job.id);
+    }
+);
+
+Then(
+    /^the PR job of "([^"]*)" is not triggered because it is restricted$/,
+    {
+        timeout: TIMEOUT
+    },
+    async function step(rootDir) {
+        // Wait 3 seconds for build trigger
+        await sdapi.promiseToWait(3);
+
+        const build = await sdapi.findBuilds(
+            {
+                instance: this.instance,
+                pipelineId: this.pipelines[rootDir].pipelineId,
+                jobName: this.jobName,
+                pullRequestNumber: this.pullRequestNumber,
+                jwt: this.jwt
+            },
+            1
```

Comment:
Is this argument is used?
https://github.com/screwdriver-cd/screwdriver/blob/master/features/support/sdapi.js#L60

Reply:
Thank you.
This is the one I forgot to erase, not used.

Reply:
I fixed it.